package com.ems.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class DashboardResponse {

    private long totalEmployees;

    private long totalDepartments;

    private long employeesHiredThisMonth;

    private BigDecimal totalPayroll;

    private BigDecimal highestSalary;

    private BigDecimal lowestSalary;

    private BigDecimal averageSalary;

}